news/2022-02-25-skinning-contest-tides-of-winter-results.md

```diff
@@ -0,0 +1,71 @@
+Within the details there lies certain textual easter-eggy touches, most of which inform the user about hotkeys and functions, while others carry background information regarding Ganyu. From the use of speech bubbles and rectangular layouts to the futuristic lines and notations, the skin gives the impression that the player is looking through a visor. In any case, if you are fond of Genshin Impact and you would like to have Ganyu assist you on osu! as well, GanyuTeks:Futur Céleste is the perfect skin for you!
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/jpK7wINmCGA" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Their beloved masterpiece GanyuTeks:Futur Céleste reserves BPChingu the 2nd place! Of course, what would be a more meaningful award than having Ganyu immortalized as their profile badge for such a delightful entry?
+
+![](/wiki/shared/news/2022-02-25-skinning-contest-tides-of-winter-results/ganyu.png)
+
+[Ganyu Artwork](https://www.pixiv.net/en/artworks/94717529) by [Doco](https://www.pixiv.net/en/users/71723824)
+
+## Winner - - 『Wintherest』 -
+
+Majestically crafted by the hands of a skinner extraordinaire, [Redo\_](https://osu.ppy.sh/users/7122165), [- 『Wintherest』 -](https://osu.ppy.sh/community/forums/topics/1498493) takes the lead with **2347** votes, surpassing the 2nd place by a short margin!
+
+Resorting to minimalist and modern styling methods, - 『Wintherest』 - brings the serenity of a wintry day and the vivid display of the aurorae together on your osu! client. The way snow is accumulated near the play button over which the logo of the skin is placed ensures a smooth transition to the anthracite layer that looks like a frosted glass filtering the lights of colour. On top of that, the use of white speech bubbles combined with various minuscule indicators strenghtens the skin's minimalistic ambience even further.
+
+Moreover, - 『Wintherest』 - not only keeps its minimalist and modern look consistent all throughout the skin, but gives the aurora effect further presence, showcased especially on the results screen. The gameplay interface on the other hand, seems to be unable to keep itself away from this aurora effect, which was placed onto the health, combo and score indicators individually, as well as the usual minimalistic aura. Without a doubt, Redo\_ did a fantastic job with this one, earning the community's approval as the most *Winteresting* pick of all.
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/8_UWmCKCTfc" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+With their - 『Wintherest』 - entry, Redo\_ gets the reward fit for a champion, a profile badge featuring the very logo of this winning masterpiece! Big congratulations to Redo\_!
```

Comment:
the "h" must be there for a reason, i'm refusing to believe otherwise

```suggestion
- 『Wintherest』 - sure did win over the rest, and for that Redo\_ gets a reward fit for a champion, a profile badge featuring the very logo of their own masterpiece! Big congratulations to Redo\_!
```